src/Admin/AbstractAdmin.php

```diff
@@ -2556,7 +2556,9 @@ public function configureActionButtons($action, $object = null)
             && $this->hasRoute('create')
         ) {
             $list['create'] = [
-                'template' => $this->getTemplateRegistry()->getTemplate('button_create'),
+                // NEXT_MAJOR: Remove this line and use commented line below it instead
+                'template' => $this->getTemplate('button_create'),
```

Comment:
Can we detect the override, trigger a deprecation if it exists and call the registry otherwise?

Reply:
A dedicated final method may be added in the admin just to do that check

Reply:
Detecting overrides in AbstractAdmin::getTemplate` may not be that easy without the use of the ReflectionClass which slows down the application.

Reply:
You're right and besides, all `getTemplate` does now is calling the registry, so nevermind my comment.